### ISSUE TITLE
Document Actions Missing in Drawer

### ISSUE THREAD

**Issue by tobiasiv** [payloadcms/payload#7429]
### Payload Version

3.0.0-beta.71

### Node Version

v20.9.0

### Next.js Version

15.0.0-canary.77

### Describe the Bug

When editing a document in a drawer, the actions to duplicate and delete are missing

### Reproduction Steps

1. Clone the [payload 3.0 demo](https://github.com/payloadcms/payload-3.0-demo)
2. Add a user relationship field to the pages collection:

```js
    {
      slug: 'pages',
      admin: {
        useAsTitle: 'title',
      },
      fields: [
        {
          name: 'title',
          type: 'text',
        },
        {
          name: 'content',
          type: 'richText',
        },
        {
          name: 'user',
          type: 'relationship',
          relationTo: 'users',
        },
      ],
    },
```
3. Update the following code from the user collection:
```js
      access: {
        delete: () => true,
        update: () => false,
      },
```
4. Run `pnpm dev`
5. Ensure that the delete action is available when editing a user outside of the drawer
6. Select a user within a pages document
7. Open the drawer to edit the user
8. The delete action is unavailable

**Comment by jmikrut** [comment]
Hey @tobiasiv — we will check into this. I believe it is intentional but I'll get with the team to double-check. We will need to brainstorm the desired UX for duplicating from a drawer, but we should be able to figure it out.

**Comment by jacobsfletch** [comment]
This was done in #7679 and has been released since [v3.0.0-beta.104](https://github.com/payloadcms/payload/releases/tag/v3.0.0-beta.104).
